pkg/cmd/cli/cmd/newapp.go

```diff
@@ -142,6 +142,7 @@ func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out io.Writer)
 		},
 	}
 
+	cmd.Flags().BoolVar(&config.AsTestDeployment, "as-test", config.AsTestDeployment, "If true create this application as a test deployment, which validates that the deployment succeeds and then scales down.")
```

Comment:
How does one indicate, on a test DC, what post-test tagging to perform?

and, given that, does it make sense to allow someone to specify that tag when they invoke new-app?
$ oc new-app myappIST --as-test --push-tag myProdAppIST

or some such syntax?  Since that's the 90% use case for this (stitching together a pipeline)?